assets/js/pages/ActivityLogPage/ActivityLogPage.jsx

```diff
@@ -1,25 +1,25 @@
-import React, { useState, useEffect } from 'react';
-import { useSearchParams } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
 import { useSelector } from 'react-redux';
+import { useSearchParams } from 'react-router-dom';
 
 import { map, pipe } from 'lodash/fp';
 
+import ActivityLogOverview from '@common/ActivityLogOverview';
```

Comment:
In order to reduce the possibility of conflicts we might want to keep the following 3 imports where they where.